.api/v1.12.0/schema.dependencies.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Please do not commit `.api ` directory to the repository as we install API automatically after npm install.
Everything you need to do is to update API version in [package.json](https://github.com/Microsoft/PowerBI-visuals-sampleBarChart/blob/61bd32aaf85f68a6013170ded2cf7f9960b1c94d/package.json#L5)